xbmc/addons/settings/AddonSettings.cpp

```diff
@@ -1341,18 +1341,19 @@ bool CAddonSettings::ParseOldLabel(const TiXmlElement* element, const std::strin
   std::string labelString;
   element->QueryStringAttribute("label", &labelString);
 
+  bool parsed = !labelString.empty();
 
   // try to parse the label as a pure number, i.e. a localized string
-  char *endptr;
-  labelId = std::strtol(labelString.c_str(), &endptr, 10);
-  if (endptr == nullptr || *endptr == '\0')
-    return true;
-
-
-  // as a last resort use the setting's identifier as a label
-  bool parsed = !labelString.empty();
-  if (!parsed)
-    labelString = settingId;
+  if (parsed)
+  {
+    char *endptr;
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
This code has just been moved / indented from above. Should I still fix it? Personally I prefer to keep syntax changes out of functional changes.

Reply:
If you prefer to keep syntax changes in a separate commit add one before the actual code changes.
IMO it just spams the commit history and you could add it to the actual code change as the line is touched anyway.

Reply:
Fixed